schematic/store/synapse.py

```diff
@@ -961,12 +959,15 @@ def format_row_annotations(self, se, sg, row, entityId, useSchemaLabel, hideBlan
                 annos.pop(anno_k) if anno_k in annos.keys() else annos
             # Otherwise save annotation as approrpriate
             else:
-                if isinstance(anno_v,float) and np.isnan(anno_v):
-                        annos[anno_k] = ""
-                elif isinstance(anno_v,str) and re.fullmatch(csv_list_regex, anno_v) and rule_in_rule_list('list', sg.get_node_validation_rules(anno_k)):
-                    annos[anno_k] = anno_v.split(",")
-                else:
-                    annos[anno_k] = anno_v
+                try:
+                    if isinstance(anno_v,float) and np.isnan(anno_v):
+                            annos[anno_k] = ""
+                    elif isinstance(anno_v,str) and re.fullmatch(csv_list_regex, anno_v) and rule_in_rule_list('list', sg.get_node_validation_rules(anno_k)):
+                        annos[anno_k] = anno_v.split(",")
+                    else:
+                        annos[anno_k] = anno_v
+                except:
+                    breakpoint()
```

Comment:
Sorry I didn't catch before, but should we raise a specific error here @mialy-defelice?

Reply:
Oops I think I had hit a bug at one point and was just debugging, this was not an actual test. I'll remove it, thanks for catching it!!